src/graphs.jl

```diff
@@ -122,6 +129,12 @@ function Graphs.add_edge!(meta_graph::MetaGraph, label_1, label_2, data)
     return added
 end
 
+function Graphs.add_edge!(meta_graph::MetaGraph{<:Any, <:Any, <:Any, <:Any, Nothing},
```

Comment:
Be careful with formatting, you need to apply `JuliaFormatter.format(MetaGraphsNext)` before pushing, otherwise the tests won't pass

Reply:
Ah, didn't know that: thanks!